pmlc/rt/vulkan/tests/CPPLINT.cfg

```diff
@@ -0,0 +1 @@
+filter=-whitespace/line_length
```

Comment:
Why is this needed?

Reply:
This is added because when I git commit, some lines of the cc_test_gpu.cc are too long.

Reply:
Oh, I see this is in the `vulkan/tests` directory, ok great.